apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -405,8 +405,8 @@ type BareMetalHostSpec struct {
 	// +kubebuilder:validation:Pattern=`[0-9a-fA-F]{2}(:[0-9a-fA-F]{2}){5}`
 	BootMACAddress string `json:"bootMACAddress,omitempty"`
 
-	// Should the host be powered on? Changing this value will trigger
-	// a change in power state of the host.
+	// Should the host be powered on? If the host is not currently being
+	// provisioned, its power state will be forced to match this value.
```

Comment:
Consider saying "once the host is in a steady state", since there are other states where this is ignored (e.g. inspecting, preparing).